roles/istio/templates/gateway.yaml.j2

```diff
@@ -0,0 +1,28 @@
+apiVersion: networking.istio.io/v1beta1
+kind: Gateway
+metadata:
+  name: seldon-gateway
+  namespace: istio-system
+spec:
+  selector:
+    app: istio-ingressgateway
+    istio: ingressgateway
+  servers:
+  - hosts:
+    - '*'
+    port:
+      name: http
+      number: 80
+      protocol: HTTP
+{%- if istio_secure %}
```

Comment:
What is `-` doing after `{%`?

Reply:
It just removes whitespace. It's similar to golang templating

You can check out an example here https://cryptic-cliffs-32040.herokuapp.com/ (used this for testing)